shylands/climatechoice
# Optimise for performance

## Done

Improve the performance of the site by optimising image assets and preconnecting to web fonts CDN.

## QA

- Browse the site and visually check each image asset for malformation, they all look good to me I may have missed any small detail.
